gradle-plugin/src/main/kotlin/com/google/devtools/ksp/gradle/KspAATask.kt

```diff
@@ -76,7 +81,11 @@ abstract class KspAATask @Inject constructor(
             val sourceSetName = kotlinCompilation.defaultSourceSet.name
             val kspTaskName = kotlinCompileProvider.name.replaceFirst("compile", "ksp")
             val kspAADepCfg = project.configurations.detachedConfiguration(
-                project.dependencies.create("${KspGradleSubplugin.KSP_GROUP_ID}:symbol-processing-aa:$KSP_VERSION")
+                project.dependencies.create("${KspGradleSubplugin.KSP_GROUP_ID}:symbol-processing-aa:$KSP_VERSION"),
+                project.dependencies.create(
+                    "${KspGradleSubplugin.KSP_GROUP_ID}:symbol-processing-gradle-plugin:$KSP_VERSION"
+                ),
+                project.dependencies.create("org.jetbrains.intellij.deps:trove4j:1.0.20200330"),
```

Comment:
no idea where this comes from, call site looks like intellij deps to me but intellij jar relocated the symbols therefore has to use this weired version.

Reply:
Shouldn't trove4j be packed into the fat jar already?

Reply:
No, the one found in fat jar does not sits in `gnu` package name, that's the relocation issue I am referring to.

Reply:
Can you please check how it is included in the compiler. If we need this, we should pack it and remove the one we already have. Having both sounds wrong to me.

Reply:
This class is relocated multiple times by different consumers, e.g. Gradle also relocated this class under its own name. What's causing trouble here is the non relocated version got involved, and is not part of any fat jar since fat jars all do relocations.